pkg/util/cryptomaterial/certchains/signers.go

```diff
@@ -459,7 +457,7 @@ func libraryGoMakeAndWriteSubCA(ca *crypto.CA, certFile, keyFile, serialFile, na
 	var serialGenerator crypto.SerialGenerator
 	if len(serialFile) > 0 {
 		// create / overwrite the serial file with a zero padded hex value (ending in a newline to have a valid file)
-		if err := os.WriteFile(serialFile, []byte("00\n"), 0644); err != nil {
+		if err := os.WriteFile(serialFile, []byte("00\n"), 0644); err != nil { //nolint:gosec
```

Comment:
In in the data dir accessible only by `root`, wdyt about `0600`?

Reply:
+1